### PR TITLE
feat(docs): keep a changelog

### PR DESCRIPTION
**Pull request by dannysheridan**

### CODE REVIEW

**Comment by chdeskur** [comment]
Preview link failing to generate

File: fern/pages/docs/writing-content/changelog.mdx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+Changelogs contained within your API folder will automatically be displayed at the bottom of your API reference. You do not need to configure your changelog(s) within `docs.yml`. 
+
+View an example of a changelog per API in [Astronomer's docs](https://www.astronomer.io/docs/api/iam-api-reference/changelog).
+
+<Frame
+    caption="A unique changelog per API"
+    background="subtle"
+  >
+      <img src="changelog-astronomer.png" />
```
Comment (chdeskur) [inline fern/pages/docs/writing-content/changelog.mdx:113]: I think the icon here looks bad. It's not centered. We should prioritize a styling fix to this next week.

File: fern/pages/docs/writing-content/changelog.mdx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+```mdx
+## Summary
+
+In the latest release, we've added endpoints to create a new Plant.
+
+### What's new?
+
+New endpoints:
+
+- `POST /plant` add a new plant to inventory.
```
[the remark this answers is not in the document]
Comment (dannysheridan) [reply fern/pages/docs/writing-content/changelog.mdx:133]: will follow up with this https://github.com/fern-api/docs-starter/pull/54

File: fern/pages/docs/writing-content/changelog.mdx (hunk 1 above)
Comment (dannysheridan) [reply fern/pages/docs/writing-content/changelog.mdx:113]: agreed, doesn't look great. out of scope for this PR though.

**Pull request merged**